domains/rogo.json

```
@@ -0,0 +1,11 @@
+{
+  "owner": {
+    "owner": "BenRogozinski",
+    "discord": "BenDaBuilder",
+    "email": "brogodev@gmail.com"
+  },
+  "record": {
+    "A": "5.161.233.190",
```

```suggestion
    "A": ["5.161.233.190"],
```
A records must be in an array.